src/sentry/interfaces/base.py

```diff
@@ -95,7 +118,7 @@ def get_path(self):
     def get_alias(self):
         return self.get_slug()
 
-    def get_hash(self):
+    def get_hash(self, **kwargs):
```

Comment:
I think we should keep this method signature explicit, *especially* in the base implementation. This way if we add another parameter in the future we can be sure it's implemented everywhere and not silently disregarded.

I think the parameter should also be documented here. (In general a docstring explaining what `get_hash` actually is doing/is supposed to do would be a good idea for our future sanity.)